Documentation on how to view dataspace catalog through the UI (#132)
agera-edc/MinimumViableDataspace: docs/developer/view-catalog.md

# View Catalog

## Overview

> Dataspace participants can use EDC data dashboard to view what all contract offers are shared with them. It is called `Catalog` and it gets refreshed frequently as new contract offers are added by other participants.

Reviewer: nit: 
```suggestion
Dataspace participants can use the EDC data dashboard to view  contract offers are shared with them. It is called `Catalog` and it gets refreshed frequently as new contract offers are added by other participants.
```

Author: Applied.

> Dataspace participants can use EDC data dashboard to view what all contract offers are shared with them. It is called `Catalog` and it gets refreshed frequently as new contract offers are added by other participants.

Reviewer 2: Q for clarity: what gets refreshed the UI or the catalog? Or is it the same?

And what should the end user expect reading `frequently`? Is there a default value?

Author: 1. Catalog gets refreshed in background and so the UI (not automatically, needs a page reload)
2. Added more information for further clarity. 

Thank you.

> Dataspace participants can use EDC data dashboard to view what all contract offers are shared with them. It is called `Catalog` and it gets refreshed frequently as new contract offers are added by other participants.

Author: Applied.

## Viewing Catalog/Offers

> * Go to `Dataspace Catalog` pane and here you will see all existing contract offers which are shared with you.

Reviewer 2: Question on `you`: Is this sharing based on an end user identity or is it based on the `dataspace participant` (= EDC connector?)?

I'm trying to understand it in the context of the introduction:
`Dataspace participants can use EDC data dashboard to view what all contract offers are shared with them.`

Author: Update for more clarity. Thank you.
